### ISSUE TITLE
A redundant line of code in transform.py

### ISSUE THREAD

**Issue by X-Lai** [nicolas-chaulet/torch-points3d#609]
Firstly, thanks a lot for your great work. 

However, I feel a little confused about this [line of code](https://github.com/nicolas-chaulet/torch-points3d/blob/master/torch_points3d/core/data_transform/transforms.py#L139).

I wonder if this line can be deleted. Or do I miss something? Thanks a lot.

**Comment by nicolas-chaulet** [comment]
Yes you are correct, looks like `ind` is never used after that line. Feel free to submit a PR and thanks for reporting!
